subt/docker/sln_bash/Dockerfile

```diff
@@ -1,6 +1,10 @@
 FROM osrf/subt-virtual-testbed:subt_solution_latest
 
-RUN /usr/bin/python3 -m pip install --user --no-cache-dir msgpack pyserial "opencv-python>3.4.2,<4" pyzmq
+ARG user_id
+RUN sudo groupmod -g ${user_id} developer
+RUN sudo usermod -u ${user_id} developer
```

Comment:
what is the motivation here?

Reply:
The idea is that the user `developer` will have the same uid and gid as the user building the image. When you later want to share files between the host and the container using volumes, it is easy, since both use the same uid and gid. The support for this is already in the base image but since we are using the prebuilt image, we could not take advantage of this feature. This brings it back.